src/azure-cli/azure/cli/command_modules/appconfig/_utils.py

```diff
@@ -86,7 +96,7 @@ def resolve_connection_string(cmd, config_store_name=None, connection_string=Non
 
     if not string:
         raise CLIError(
-            'Please specify config store name or connection string(suggested).')
+            'If you are using "key" auth mode, please specify config store name or connection string(suggested).')
     return string
```

Comment:
No change needed. Previously this method is called almost for every single command to finalized the connection string. As we start to support aad, this one should only be invoked when resolving connection string is needed. So it must be under “key” auth mode context.